PROJ-21 - Wireframes (#8)
SW805F20/report: sections/sprint-1/prototypes.tex

\section{Prototypes}
To come to a common vision about the layout and functionality of the system, a series of prototypes were created.
The primary focus of these prototypes is not to be used for implementation, but rather for comparing opinions about how the flow in the system should be created.

> Since the user interface is mostly focused on the mobile devices that the players will be wearing, it was decided that the host computer should simply be provided by a text-based interface, as seen on \autoref{fig:prototype:hostlobby}.

Reviewer: ```suggestion
Since the user interface is mostly focused on the mobile devices that the players will be wearing, it was decided that the host computer should simply have a text-based interface, as seen on \autoref{fig:prototype:hostlobby}.
```
